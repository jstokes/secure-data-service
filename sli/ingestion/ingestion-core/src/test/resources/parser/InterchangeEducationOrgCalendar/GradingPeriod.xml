<?xml version="1.0" encoding="UTF-8"?>
<InterchangeEducationOrgCalendar xmlns:sli="http://slc-sli/ed-org/0.1" xmlns="http://ed-fi.org/0100"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <GradingPeriod id="ID009">
    <GradingPeriod>Sixth Six Weeks</GradingPeriod>
    <SchoolYear>2009-2010</SchoolYear>
    <EducationOrganizationReference>
      <EducationalOrgIdentity>
        <StateOrganizationId>J..1bKVFuQV4yn7xK</StateOrganizationId>
      </EducationalOrgIdentity>
    </EducationOrganizationReference>
    <BeginDate>2014-04-22</BeginDate>
    <EndDate>2013-05-01</EndDate>
    <TotalInstructionalDays>1108330730</TotalInstructionalDays>
    <CalendarDateReference>
      <CalendarDateIdentity>
        <Date>2014-06-09</Date>
        <EducationOrgReference>
          <EducationalOrgIdentity>
            <StateOrganizationId>gg0ssCxSVgFHrRvxVDQST8TX.raagr634T1HIU9Bbs_dU3cORd97wS9aE.</StateOrganizationId>
          </EducationalOrgIdentity>
        </EducationOrgReference>
      </CalendarDateIdentity>
    </CalendarDateReference>
    <CalendarDateReference>
      <CalendarDateIdentity>
        <Date>2013-12-26</Date>
        <EducationOrgReference>
          <EducationalOrgIdentity>
            <StateOrganizationId>Q_</StateOrganizationId>
          </EducationalOrgIdentity>
        </EducationOrgReference>
      </CalendarDateIdentity>
    </CalendarDateReference>
  </GradingPeriod>
</InterchangeEducationOrgCalendar>
