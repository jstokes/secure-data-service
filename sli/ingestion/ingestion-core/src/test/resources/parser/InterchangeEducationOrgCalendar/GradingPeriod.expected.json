{
  "@id":"ID009",
  "GradingPeriod":{
    "_value":"Sixth Six Weeks"
  },
  "SchoolYear":{
    "_value":"2009-2010"
  },
  "EducationOrganizationReference":{
    "EducationalOrgIdentity":{
      "StateOrganizationId":{
        "_value":"J..1bKVFuQV4yn7xK"
      }
    }
  },
  "EndDate":{
    "_value":"2013-05-01"
  },
  "BeginDate":{
    "_value":"2014-04-22"
  },
  "CalendarDateReference":[
    {
      "CalendarDateIdentity":{
        "Date":{
          "_value":"2014-06-09"
        },
        "EducationOrgReference":{
          "EducationalOrgIdentity":{
            "StateOrganizationId":{
              "_value":"gg0ssCxSVgFHrRvxVDQST8TX.raagr634T1HIU9Bbs_dU3cORd97wS9aE."
            }
          }
        }
      }
    },
    {
      "CalendarDateIdentity":{
        "Date":{
          "_value":"2013-12-26"
        },
        "EducationOrgReference":{
          "EducationalOrgIdentity":{
            "StateOrganizationId":{
              "_value":"Q_"
            }
          }
        }
      }
    }
  ],
  "TotalInstructionalDays":{
    "_value":"1108330730"
  }
}
